src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
+  text-align: left;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf) format(ttf);
+  src: url(/src/fonts/Avenir-Heavy.ttf) format(ttf);
+  src: url(/src/fonts/Avenir.ttc) format(ttc);
```

Comment:
The format for TrueType Collection fonts should be `format('truetype-collection')`, not `format(ttc)`.